src/appkit.cs

```diff
@@ -13344,7 +13344,7 @@ interface NSSliderTouchBarItem
 		NSSliderAccessory MaximumValueAccessory { get; set; }
 
 		[Export ("valueAccessoryWidth")]
-		double ValueAccessoryWidth { get; set; }
+		nfloat ValueAccessoryWidth { get; set; }
```

Comment:
@timrisi  this needs to get into `cycle9` before it's released. Please prepare another PR for this change. Thanks!